app/models/pet.rb

```diff
@@ -10,4 +12,8 @@ def shelter_name
   def self.adoptable
     where(adoptable: true)
   end
+
+  def self.adoptable_search(search_params)
```

Comment:
This search method is actually already implemented for you in the ApplicationRecord! This is a bit of an Easter Egg (although you're not penalized for not finding it!). Because this is a brownfield project, we want you to explore the codebase and leverage the methods already created (like you did with  the `error_messages` helper method in ApplicationController). So, you could've used this `.search` method from the model parent class and just tested it in order to complete this user story